IOUtils is broken on Android 6 and Android 7

```
java.lang.NoSuchMethodError: 
  at org.apache.commons.io.IOUtils.<clinit> (IOUtils.java:183)
  at org.apache.commons.io.IOUtils.toString (IOUtils.java:2838)
  at de.tutao.tutanota.push.TutanotaNotificationsHandler.executeMissedNotificationDownload (TutanotaNotificationsHandler.java:155)
  at de.tutao.tutanota.push.TutanotaNotificationsHandler.downloadMissedNotification (TutanotaNotificationsHandler.java:87)
  at de.tutao.tutanota.push.TutanotaNotificationsHandler.onNewNotificationAvailable (TutanotaNotificationsHandler.java:51)
  at de.tutao.tutanota.push.PushNotificationService$1.onMessage (PushNotificationService.java:66)
  at de.tutao.tutanota.push.SseClient.handleLine (SseClient.java:180)
  at de.tutao.tutanota.push.SseClient.connect (SseClient.java:115)
  at de.tutao.tutanota.push.SseClient.$r8$lambda$Yc-1qMBtLmxFXBrmqRgv1fRITQs (SseClient.java)
  at de.tutao.tutanota.push.SseClient$$ExternalSyntheticLambda1.run (Unknown Source)
  at android.os.Handler.handleCallback (Handler.java:836)
  at android.os.Handler.dispatchMessage (Handler.java:103)
  at android.os.Looper.loop (Looper.java:203)
  at de.tutao.tutanota.push.LooperThread.run (LooperThread.java:22)
```

```
java.lang.NoClassDefFoundError: 
  at org.apache.commons.io.IOUtils.closeQuietly (IOUtils.java:571)
  at de.tutao.tutanota.Crypto.aesDecrypt (Crypto.java:385)
  at de.tutao.tutanota.Crypto.aesDecrypt (Crypto.java:340)
  at de.tutao.tutanota.Crypto.aesDecrypt (Crypto.java:298)
  at de.tutao.tutanota.alarms.AlarmInfo.getTriggerDec (AlarmInfo.java:32)
  at de.tutao.tutanota.alarms.AlarmNotificationsManager.schedule (AlarmNotificationsManager.java:112)
  at de.tutao.tutanota.alarms.AlarmNotificationsManager.reScheduleAlarms (AlarmNotificationsManager.java:51)
  at de.tutao.tutanota.push.PushNotificationService.onCreate (PushNotificationService.java:55)
  at android.app.ActivityThread.handleCreateService (ActivityThread.java:3328)
  at android.app.ActivityThread.-wrap5 (ActivityThread.java)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:1659)
  at android.os.Handler.dispatchMessage (Handler.java:105)
  at android.os.Looper.loop (Looper.java:156)
  at android.app.ActivityThread.main (ActivityThread.java:6517)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:942)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:832)
Caused by: java.lang.NoSuchMethodError: 
  at org.apache.commons.io.IOUtils.<clinit> (IOUtils.java:183)
  at org.apache.commons.io.IOUtils.copyLarge (IOUtils.java)
  at de.tutao.tutanota.Crypto.aesDecrypt (Crypto.java:358)
```

Results in constant crashes for background process and crashes the whole app when downloading files.

--- a/app-android/app/build.gradle
+++ b/app-android/app/build.gradle
@@ -116,7 +116,8 @@ dependencies {
 	androidTestImplementation('com.android.support.test.espresso:espresso-core:2.2.2', {
 		exclude group: 'com.android.support', module: 'support-annotations'
 	})
-	implementation 'commons-io:commons-io:2.11.0'
+	// Important: cannot be updated without additional measures as Android 6 and 7 do not have Java 9
+	implementation 'commons-io:commons-io:2.5'
 	implementation 'org.jdeferred:jdeferred-core:1.2.4'
 	implementation 'androidx.core:core:1.7.0'
 	implementation "androidx.activity:activity:$activity_version"

